ncl/receipts/trigonometric_functions.ipynb

```diff
@@ -0,0 +1,275 @@
+{
```

Comment:
<p>Since this is the NCL receipt, should these be the NCL function names?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NCAR/geocat-applications/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>